pkg/manifests/config.go

```diff
@@ -38,6 +38,19 @@ type Config struct {
 	UserWorkloadConfiguration      *UserWorkloadConfiguration      `json:"-"`
 }
 
+func (c Config) IsStorageConfigured() bool  {
+	if c.ClusterMonitoringConfiguration == nil {
+		return false
+	}
+
+	prometheusK8sConfig := c.ClusterMonitoringConfiguration.PrometheusK8sConfig
+	if prometheusK8sConfig == nil {
+		return false
+	}
+
+	return prometheusK8sConfig.VolumeClaimTemplate != nil
```

Comment:
What if PersistentVolumeClaim itself fails? I'm not sure whether that has been already handled.

Reply:
I think this is something to be monitored with an alert and is not really in the scope of what this PR tries to achieve.

Reply:
If the PVC can't be fulfilled (wrong storageclass for instance), the prometheus pods would never start and we should get an alert from CMO.